tl/cli/context_match.py

```diff
@@ -0,0 +1,41 @@
+import typing
+import argparse
+import sys
+import traceback
+import tl.exceptions
+
+
+def parser():
+    return {
+        'help': 'Match the context values to the properties and add the score of the match.'
+    }
+
+
+def add_arguments(parser):
+    from tl.utility.utility import Utility
+    from kgtk.cli.text_embedding import ALL_EMBEDDING_MODELS_NAMES
+    # input file
+    parser.add_argument('input_file', nargs='?', type=argparse.FileType('r'), default=sys.stdin)
+    # debug
+    parser.add_argument('context_file', nargs='?', type=argparse.FileType('r'), default=sys.stdin)
+    parser.add_argument('--debug', action='store_true', help="if set, an kgtk debug logger will be saved at home directory. Debug adds two new columns to the output denoting the properties matched and the respective similarities.")
+    parser.add_argument('--sim_string', action='store', type=float, dest='sim_string', default=0.75,
+                        help='The minimum threshold for similarity with input context for string matching.')
+    parser.add_argument('--sim_quantity', action='store', type=float, dest='sim_quantity', default=0.85,

```

Comment:
rename this option to make more sense as well. I cant figure out what this means, change `dest` as well

Reply:
Hi, sim_quantity is the threshold for matching the wikidata quantities. Since, in some of the files, we have quantities as well apart from strings in the context.